lms/djangoapps/bulk_email/models.py

```diff
@@ -470,6 +470,27 @@ def __str__(self):
             not_en = ""
         return f"Course '{str(self.course_id)}': Instructor Email {not_en}Enabled"
 
+class DisabledCourse(models.Model):
+    """
+    Disable the bulk email feature for specific courses.
+
+    .. no_pii:
+    """
+
+    course_id = CourseKeyField(max_length=255, db_index=True, unique=True)
+    email_disabled = models.BooleanField(default=False)
+
+    @classmethod
+    def instructor_email_disabled_for_course(cls, course_id):
```

Comment:
What is `cls`? Can we name this something less ambiguous, or else annotate with a comment?

Reply:
I _think_ this is pretty standard first-argument for Python class methods... python.org/dev/peps/pep-0008/#function-and-method-arguments.

Reply:
(Never mind! This seems to be standard practice for Django to represent the `class` itself in `@classmethod`s. No change here.)

Reply:
(Oh ha! Sorry Justin, I hadn’t refreshed and didn’t see your comment.)